src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,92 @@
-import React from 'react';
+import React, {
+  useState,
+} from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
+import { defaultPosterLink } from '../../constants';
+
+type Props = {
+  movies: Movie[],
+  setMovies: (movies: Movie[]) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [title, setTitle] = useState<string>('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
  const [movie, setMovie] = useState<Movie | null>(null);
  const [isLoading, setIsLoading] = useState(false);
  const [isError, setIsError] = useState(false);
```
it's redundant to specify primitive types